resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -5,10 +5,12 @@ restartfile=<%= EZBake::Config[:restart_file] %>
 timeout=<%= EZBake::Config[:start_timeout] %>
 
 if [ -r "$restartfile" ];  then
-    kill -HUP $(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
+    pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
+    kill -HUP $pid
     cur=$(head -n 1 "$restartfile")
     initial=$cur
     while [ "$cur" == "$initial" ] ;do
+        kill -0 $pid >/dev/null 2>&1
```

Comment:
Not sure if `kill -0 $pid` is the best approach for ensuring that a pid is still active but found several references on the webz - e.g., http://stackoverflow.com/questions/11012527/what-does-kill-0-pid-in-a-shell-script-do and http://stackoverflow.com/questions/13260974/how-to-check-if-process-is-running-in-linux - indicating that it is a reasonably portable approach.  It seems to work fine on the different OSes I tested - EL6 / 7, Ubuntu 12.04 / 14.04 / 16.04, SLES 11 / 12.  Happy to use something different if others have better suggestions.